ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/representative-journey.spec.ts

```diff
@@ -14,6 +15,9 @@ dayAfterTomorrow.setDate(tomorrow.getDate() + 2);
 describe('RepresentativeJourney', () => {
   let journey: RepresentativeJourney;
   let redirected: JourneyStep;
+  let submitService: jasmine.SpyObj<SubmitService>;
+  submitService = jasmine.createSpyObj<SubmitService>(['submit', 'isDropOffPoint', 'updateSubmitModel']);
```

Comment:
It might be alright in this instance but watch out for assigning the service mocks outside of `beforeEach` because all tests will use the same instance if it is this way. This means that mocks set up in one test could affect the outcome of another test.

Reply:
Will do